### PR TITLE
fix(index): add default export for parser

### LINKED ISSUE #143: Index.d.ts does not match with Index.js
Is this a bug report?
> Yes

## Expected Behavior
Default import in es modules should have a function defined.

## Actual Behavior
Default import is `undefined`.

### PR DESCRIPTION
**Pull request by heldergoncalves92**
Fixes #143

- [x] Passing [build](https://travis-ci.org/github/remarkablemark/html-react-parser/builds/661639738)
- [x] Add test

### CODE REVIEW

**Comment by heldergoncalves92** [comment]
> Thanks for the opening the PR!
> 
> Do you mind amending your commit so the [build](https://travis-ci.org/github/remarkablemark/html-react-parser/builds/661639738) passes?
> 
> ```shell
> git commit --amend -m "fix(index): add default export for parser"
> git push -f
> ```
> 
> Also, do you mind adding a test?

I can add it! 👍

File: test/html-to-react.js [hunk 1]
```diff
@@ -13,6 +16,10 @@ describe('html-to-react', () => {
       });
     });
 
+    it('has default ES Module defined', () => {
+      assert.notEqual(ESParser['default'], undefined);
```
Comment (remarkablemark) [inline test/html-to-react.js]: Nit: you can remove lines 6-8 and update test to:
```js
assert.strictEqual(parse.default, parse);
```

**Comment by remarkablemark** [comment]
Also minor note, the last commit message could be:
```
test(html-to-react): add test to ensure default export for parser
```

We use [conventional commits](https://www.conventionalcommits.org/) to release, so it follows the [Angular commit message guidelines](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines).

File: test/html-to-react.js (hunk 1 above)
Comment (heldergoncalves92) [reply test/html-to-react.js]: All right!

**Pull request merged**